Libs/MRML/DisplayableManager/vtkMRMLSliceIntersectionInteractionRepresentation.h

```diff
@@ -189,7 +189,7 @@ class VTK_MRML_DISPLAYABLEMANAGER_EXPORT vtkMRMLSliceIntersectionInteractionRepr
     class vtkInternal;
     vtkInternal* Internal;
 
-    vtkMRMLSliceIntersectionInteractionRepresentationHelper* Helper;
+    vtkSmartPointer<vtkMRMLSliceIntersectionInteractionRepresentationHelper> Helper;
```

Comment:
For future reference, you can do: `vtkNew<vtkMRMLSliceIntersectionInteractionRepresentationHelper> Helper;` in the header file and then you don't need to do anything in the cxx file.